diff --git a/node_modules/@rhds/elements/elements/rh-stat/rh-stat.css.js b/node_modules/@rhds/elements/elements/rh-stat/rh-stat.css.js
index c965204..f492190 100644
--- a/node_modules/@rhds/elements/elements/rh-stat/rh-stat.css.js
+++ b/node_modules/@rhds/elements/elements/rh-stat/rh-stat.css.js
@@ -26,11 +26,9 @@ span,
   text-align: center;
 }
 
-pfe-icon[size="md"],
-::slotted(pfe-icon[size="md"]) {
-  --pfe-icon--size: var(--rh-font-size-heading-xl, 2.5rem);
-
-  display: contents;
+.hasIcon #icon {
+  width: var(--rh-size-icon-04, 40px);
+  height: var(--rh-size-icon-04, 40px);
 }
 
 .hasIcon ::slotted([slot="icon"]),
@@ -39,6 +37,9 @@ pfe-icon[size="md"],
   display: block;
   color: black;
   margin-bottom: 16px;
+  width: 100%;
+  height: 100%;
+  --pfe-icon--size: 100%;
 }
 
 #title {
diff --git a/node_modules/@rhds/elements/rhds.min.js b/node_modules/@rhds/elements/rhds.min.js
index 2899a78..17e2320 100644
--- a/node_modules/@rhds/elements/rhds.min.js
+++ b/node_modules/@rhds/elements/rhds.min.js
@@ -3863,11 +3863,9 @@ span,
   text-align: center;
 }
 
-pfe-icon[size="md"],
-::slotted(pfe-icon[size="md"]) {
-  --pfe-icon--size: var(--rh-font-size-heading-xl, 2.5rem);
-
-  display: contents;
+.hasIcon #icon {
+  width: var(--rh-size-icon-04, 40px);
+  height: var(--rh-size-icon-04, 40px);
 }
 
 .hasIcon ::slotted([slot="icon"]),
@@ -3876,8 +3874,12 @@ pfe-icon[size="md"],
   display: block;
   color: black;
   margin-bottom: 16px;
+  width: 100%;
+  height: 100%;
+  --pfe-icon--size: 100%;
 }
 
+
 #title {
   order: 1;
   color: var(--_accent-color);
